engine/src/main/scala/cromwell/webservice/routes/wes/WesRouteSupport.scala

```diff
@@ -0,0 +1,89 @@
+package cromwell.webservice.routes.wes
+
+import akka.actor.ActorRef
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import akka.pattern.{AskTimeoutException, ask}
+import akka.util.Timeout
+import cromwell.engine.instrumentation.HttpInstrumentation
+import cromwell.services.metadata.MetadataService.{GetStatus, MetadataServiceResponse, StatusLookupFailed, StatusLookupResponse}
+import cromwell.webservice.routes.CromwellApiService.{EnhancedThrowable, UnrecognizedWorkflowException, validateWorkflowId}
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+import WesResponseJsonSupport._
+import akka.http.scaladsl.model.{StatusCode, StatusCodes}
+import akka.http.scaladsl.server.directives.RouteDirectives.complete
+import WesRouteSupport._
+import cromwell.core.WorkflowId
+import cromwell.engine.workflow.WorkflowManagerActor.WorkflowNotFoundException
+import cromwell.server.CromwellShutdown
+import cromwell.webservice.routes.CromwellApiService
+
+trait WesRouteSupport extends HttpInstrumentation {
+  val serviceRegistryActor: ActorRef
+  val workflowManagerActor: ActorRef
+  val workflowStoreActor: ActorRef
+
+  implicit val ec: ExecutionContext
+  implicit val timeout: Timeout
+
```

Comment:
add a comment here explaining the URL structure in relation to the primary cromwell endpoints

Reply:
done